Core/NoOpRunner.Core/Enums/PlayerOneState.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace NoOpRunner.Core.Enums
+{
+    public enum PlayerOneState
+    {
+        IdleState,
```

Comment:
No need for state in the enums, it's already expressed in the enum name

Reply:
Yes